server/store/middleware/response/response.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 The Chihaya Authors. All rights reserved.
+// Use of this source code is governed by the BSD 2-Clause license,
+// which can be found in the LICENSE file.
+
+package response
+
+import (
+	"github.com/chihaya/chihaya"
+	"github.com/chihaya/chihaya/server/store"
+	"github.com/chihaya/chihaya/tracker"
+)
+
+func init() {
+	tracker.RegisterAnnounceMiddleware("store_response", responseAnnounceClient)
+	tracker.RegisterScrapeMiddleware("store_response", responseScrapeClient)
+}
+
+// ErrCouldntGetPeers is returned when something goes wrong with getting
+// peers from the store.
+var ErrCouldntGetPeers = tracker.ClientError("couldn't get peers")
```

Comment:
Maybet this should actually be a type instead of a variable. That way we can wrap the actual error we get with this type and not loose any context.